DigitalLearningSolutions.Web/Styles/frameworks/frameworksShared.scss

```diff
@@ -138,54 +37,34 @@ div#search {
   }
 }
 
-.sort-row:not(.no-label)::before {
-  content: "Sort by:";
-  position: relative;
-  top: -32px;
-  height: 0;
-  width: 0;
-  float: left;
-  font-weight: 600;
-  font-size: 19px;
-  color: rgb(33, 43, 50);
-  font-family: "Frutiger W01", Arial, sans-serif;
-  white-space: nowrap;
-
-  @include mq($from: desktop) {
-    left: 8px;
-  }
-}
-
-.sort-row .nhsuk-button {
-  margin-top: 0;
-  margin-bottom: 0;
+.tags {
+  display: flex;
+  margin-bottom: 16px;
+  flex-wrap: wrap;
 }
 
-.sort-button-container {
-  padding-top: 8px;
-
-  @include mq($from: desktop) {
-    padding-top: 0;
-    padding-left: 8px;
-  }
+.tag {
+  background-color: $nhs-dark-grey;
+  color: white;
+  padding: 4px;
+  border-radius: 4px;
+  margin-right: 4px;
+  margin-bottom: 4px;
 }
 
-.nhsuk-select {
-  border-radius: 4px;
+.nhsuk-tag.role-tag {
+  min-width: 80px;
+  text-align: center;
+  font-size: 12px;
+  padding: 4px;
 }
 
 .nhsuk-details__text {
   padding-bottom: 0 !important;
 }
 
-.sort-and-search {
-  margin-top: 32px;
-  display: flex;
-  flex-direction: column;
-
-  @include mq($from: desktop) {
-    flex-direction: row;
-  }
+.nhsuk-details__text {
+  padding-bottom: 0 !important;
 }
```

Comment:
How is this different from L62?

Reply:
It's not, removed it but the frameworks css file is a mess of styles copied from elsewhere.